windows/internal/setup.bat

```diff
@@ -59,6 +59,10 @@ move /Y torch\test\*.* libtorch\test\
 
 move /Y libtorch\bin\*.dll libtorch\lib\
 
+pushd pytorch

```

Comment:
Sorry, I didn't review it carefully. The pushd/popd pair is not necessary here because we are already under the `pytorch` directory.

Reply:
I am sorry too.
Remove the pushd/popd
https://github.com/pytorch/builder/pull/276